src/styles/blocks/stars.scss

```
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url("../images/star.svg") no-repeat center;
+    height: $star-size;
+    width: $star-size;
+  }
+
+  @for $index from 1 through 5 {
+    &--#{$index} &__star:nth-child(-n+#{$index}) {
+      background: url("../images/star-active.svg") no-repeat center;
+    }
+  }
```

It's great that you have used a loop for this, I love to see it.
But keep in mind that you want to change only one property here
```suggestion
  @for $index from 1 through 5 {
    &--#{$index} &__star:nth-child(-n+#{$index}) {
      background-image: url("../images/star-active.svg");
    }
  }
```